lib/registrant.rb

```diff
@@ -0,0 +1,31 @@
+class Registrant
+    attr_reader :age,
+                :license_data,
+                :name,
+                :permit
+
```

Comment:
I don't think you need `permit` listed here since you never need to call `.permit` on a Registrant object.